### PR TITLE
feat(financial-aid): Bootstrap the project

### PR DESCRIPTION
**Pull request by addi**
Sorry about the huge PR, this project has been in a political limbo for a two months.

You should only have to review the core files: nx, workspace, tsconfig.base & codeowners.

## What

Bootstrap for the financial aid project

## Why

We want to try out the deployment pipelines 

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: workspace.json [hunk 1]
```diff
@@ -3364,6 +3364,425 @@
...
     },
+    "financial-aid-api": {
+      "root": "apps/financial-aid/api",
+      "sourceRoot": "apps/financial-aid/api/src",
+      "projectType": "application",
+      "prefix": "financial-aid-api",
+      "schematics": {},
+      "architect": {
+        "build": {
+          "builder": "@nrwl/node:build",
```
Comment (eirikurn) [inline workspace.json:3375]: Should use new esbuild setup

File: workspace.json [hunk 2]
```diff
@@ -3364,6 +3364,425 @@
...
+    },
+    "financial-aid-backend": {
+      "root": "apps/financial-aid/backend",
+      "sourceRoot": "apps/financial-aid/backend/src",
+      "projectType": "application",
+      "prefix": "financial-aid-backend",
+      "schematics": {},
+      "architect": {
+        "build": {
+          "builder": "@nrwl/node:build",
```
Comment (eirikurn) [inline workspace.json:3473]: Same here

File: workspace.json (hunk 1 above)
Comment (addi) [reply workspace.json]: @eirikurn Fixed, pls rereview.

**Review by MargretFinnboga — APPROVED** [review]
Wow (Y)

File: libs/financial-aid/auth/src/lib/auth.module.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import { SharedAuthService } from './auth.service'
+import { COOKIE_EXPIRES_IN_SECONDS } from '@island.is/financial-aid/shared'
+
+export interface SharedAuthModuleOptions {
+  jwtSecret: string
+  secretToken: string
+}
+
+@Global()
+export class SharedAuthModule {
```
Comment (eirikurn) [inline libs/financial-aid/auth/src/lib/auth.module.ts:15]: This is a lot of security-related duplicated code.

Can you use `libs/auth-nest-tools` and `libs/auth/react` instead?

**Review by eirikurn — APPROVED** [review]

File: libs/financial-aid/auth/src/lib/auth.module.ts (hunk 3 above)
Comment (addi) [reply libs/financial-aid/auth/src/lib/auth.module.ts:15]: I had a meeting with a developer from Fuglar yesterday and are having a meeting with the PO of the identity server next week. To figure out when its going to be a good time for us to migrate over.

Comment (eirikurn) [reply libs/financial-aid/auth/src/lib/auth.module.ts:15]: Ok sure. I'd like to know more, but I guess this can wait till fall :)

**Pull request merged**